lib/logic/entities/Workflow.js

```diff
@@ -6,7 +6,7 @@ class Workflow extends Entity {
         this.entityType = 'workflow';
         this.info = data;
         this.defaultColumns = ['id', 'pipeline', 'status', 'created', 'finished', 'totalTime', 'trigger', 'repository', 'branch'];
-        this.wideColumns = this.defaultColumns.concat([]);
+        this.wideColumns = this.defaultColumns.concat(['pipeline_Id']);
```

Comment:
rename field to be with - instead of _
rename pipeline => pipeline-name
put pipeline id next to pipeline name